ratings/schema.sql

```diff
@@ -29,16 +29,18 @@ CREATE TABLE IF NOT EXISTS models (
   black_wins integer,
   white_games integer,
   white_wins integer,
-  unique(bucket, model_name)
+
+  UNIQUE(bucket, model_name)
 );
 
 CREATE TABLE IF NOT EXISTS wins (
     game_id integer primary key,
     model_winner integer not null,
     model_loser integer not null,
-    foreign key(game_id) references games(game_id),
```

Comment:
references seems pointless without ON DELETE CASCADE.

also sqlite doesn't enforce these by default which is annoying